src/main/java/com/kobylynskyi/graphql/codegen/mapper/GraphQLTypeMapper.java

```diff
@@ -330,5 +318,14 @@ default String getTypeConsideringPrimitive(MappingContext mappingContext,
         return computedTypeName;
     }
 
+    default DeprecatedDefinition getDeprecated(MappingContext mappingContext, DirectivesContainer<?> directivesContainer) {
+        return directivesContainer.getDirectives().
```

Comment:
Please not mix-up leading dots and trailing dots in the chained method calls.
(across the project, we are using the leading dot)